framework/test/jest/integration/specs/modules/chain/blocks/blocks/on_receive_block.spec.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+describe('integration test (blocks) - receiveBlockFromNetwork()', () => {
+	describe('receiveBlockFromNetwork (empty transactions)', () => {
+		describe('for valid block', () => {
```

Comment:
Maybe change to `when the block is valid it should be added to the chain` and make it an `it` to avoid one `describe`?